app/js/components/filterComponents/incidentsTriState/incidentsTriState.controller.js

```diff
@@ -49,7 +57,7 @@ function incidentsTriStateCtrl($location, $rootScope, $scope, Events, IncidentFi
 
     function init () {
         $scope.showIncidents = $location.search()[Events.filters.incident] ?
-            $location.search()[Events.filters.incident] : 'all';
+            $location.search()[Events.filters.incident] : FilterService.getIncidents();
```

Comment:
since FilterService is already setting incidents to either $location.search()[Events.filters.incident] or 'all' you could shorten this to just $scope.showIncidents = FilterService.getIncidents();